certbot-dns-cloudflare/certbot_dns_cloudflare/_internal/dns_cloudflare.py

```diff
@@ -151,9 +174,14 @@ def _find_zone_id(self, domain):
                 hint = None
 
                 if code == 6003:
-                    hint = 'Did you copy your entire API key?'
+                    hint = ('Did you copy your entire API token/key? To use Cloudflare tokens, '
+                            'you\'ll need python-cloudflare>=2.3.1. You have python-cloudflare {}'
+                            .format(CloudFlare.__version__))
```

Comment:
I thought it was worth having as the version may not have caused that error, it may have been the length of the token/key, which is the reason that this error happens when using an older version of cloudflare. An older version would present a token as the global key to the server, but it's the wrong length.